src/openrct2/ui/UiContext.h

```diff
@@ -113,6 +113,7 @@ namespace OpenRCT2
             virtual void TriggerResize() abstract;
 
             virtual void ShowMessageBox(const std::string& message) abstract;
+            virtual int ShowMenuDialog(const std::vector<std::string>& options, const std::string& title, const std::string& text) abstract;
```

Comment:
Can you add a docblock here? It is not clear from the context why it returns an integer and what it means.